README.md

```diff
@@ -1,49 +1,64 @@
+-   Update to the latest release via the eXist-db package manager or via the eXist-db.org public app repository at <http://exist-db.org/exist/apps/public-repo/>.
 
 ## Building from source
 
-1. Dependencies: Maven 3.x
-
-2. Clone the repository to your system:
+1.  Dependencies: Maven 3.x
 
+2.  Clone the repository to your system:
 ```bash
 $ git clone https://github.com/exist-db/documentation.git exist-documentation
 ```
 
-3. Build the documentation application:
+3.  Build the documentation application:
 ```bash
 $ cd exist-documentation
 $ mvn clean package
 ```
 
-4. An EXPath Application Package (.xar file) is deposited in the `target` directory
+4.  An EXPath Application Package (.xar file) is deposited in the `target` directory
+
+5.  Install this file via the Dashboard > Package Manager.
 
-5. Install this file via the Dashboard > Package Manager.
+Find an area of the documentation that needs to be improved? Please raise an [issue](https://github.com/eXist-db/documentation/issues) or submit a [pull request](https://github.com/eXist-db/documentation/pulls)!
 
-Find an area of the documentation that needs to be improved? Please raise an issue and submit a pull request!
+Our test-suite performs a validation check on all articles in the `/data` directory when you open a pull request. You can speed up the review process by validating your edits locally before submitting a pull request.
 
 
 ## Building a Release from source
 
-1. Follow the instructions from [Building from source](#building-from-source)
+1.  Merge outstanding and reviewed PRs.
+
+2.  Set version numbers in `pom.xml`.
+  2. Keep in mind, that the major version of the documentation app mirrors the [major version](https://github.com/eXist-db/exist/blob/develop/exist-versioning-release.md#versioning-scheme) the current eXist release. So for exist-db v3.x.x the documentation app will be v3.y.y. Minor and Patch numbers can vary between the two.
+  3. You should also check and set the appropriate `exist.version` to prevent users of older exist releases, from installing the wrong documentation locally.
 
-2. Create a Release:
+2.  Follow the instructions from [Building from source](#building-from-source)
 
+3.  Create a Release:
 ```bash
 $ mvn release:prepare
 $ mvn release:perform
 ```
+This will create an EXPath Application Package (.xar file) in the `/target` directory
+
+5.  If you are a core contributor, you should then commit and push.
+
+  This  automatically increments the version number in the  master branch to the next `-SNAPSHOT`.
+
+6. Check [GitHub releases](https://github.com/eXist-db/documentation/releases)
+  1. The compiled .xar file of the stable release should have been  uploaded for archiving.
 
-3. An EXPath Application Package (.xar file) is deposited in the `target` directory
+  2. Edit the release notes describing which article changed and how.  
```

Comment:
It would seem to me that this should be done before `mvn release-prepare` otherwise they wont be in the Xar

Reply:
this is talking about GitHub release notes, but the indenting got cobbled up. I can add another step for the xar release notes though

Reply:
ah ok